src/scripts/main.js

```diff
@@ -1,3 +1,49 @@
 'use strict';
 
-// write your code here
+function promise1() {
```

Comment:
you can simplify this function to be just:

```
const promise1 = new Promise((resolve, reject) => {
  document.querySelector('.logo').addEventListener('click', resolve);
});
```

and move logic for creating HTML in to separate function which will called on resolve status